storage/storage.cpp

```diff
@@ -601,9 +600,11 @@ void Storage::OnServerListDownloaded(vector<string> const & urls)
   TCountryId const & countryId = queuedCountry.GetCountryId();
   MapOptions const file = queuedCountry.GetCurrentFile();
 
+  vector<string> const downloadingUrls =
```

Comment:
Do you really need a copy here, or const reference will be enough? BTW, why not to use auto const & here?